crates/sui-graphql-rpc/Cargo.toml

```diff
@@ -51,6 +51,7 @@ tap.workspace = true
 telemetry-subscribers.workspace = true
 tracing.workspace = true
 tokio.workspace = true
+tokio-util = { workspace = true, features = ["rt"] }
```

Comment:
From what I can tell, the `rt` flag enables the "current-thread" scheduler:

https://docs.rs/tokio/latest/tokio/runtime/index.html#runtime-configurations

I know this is the `tokio-util` crate, but it seems that `rt` on this crate implies `rt` on the `tokio` crate:

https://docs.rs/crate/tokio-util/0.7.10/features

I have not seen the rest of the PR, so maybe this is OK, but my initial thought is that I would expect us to need to use a multi-threaded runtime to run the tasks we want.

Reply:
My understanding is that tokio defaults to the multi-threaded runtime unless one explicitly uses the single or current-thread schedule, by setting the flavor on `tokio::main` or using the runtime builder manually. Specifying the `rt` feature flag exposes the `new_current_thread` but doesn't lock us in to that behavior

Also going off the docs here for gracefully shutting down using tokio: https://docs.rs/tokio-util/0.7.10/tokio_util/task/task_tracker/struct.TaskTracker.html#wait-for-tasks-to-exit